lib/jekyll/entry_filter.rb

```diff
@@ -1,27 +1,30 @@
 module Jekyll
   class EntryFilter
-    SPECIAL_LEADING_CHARACTERS = ['.', '_', '#', '~'].freeze
-
     attr_reader :site
+    SPECIAL_LEADING_CHARACTERS = [
+      '.', '_', '#', '~'
+    ].freeze
 
     def initialize(site, base_directory = nil)
       @site = site
-      @base_directory = derive_base_directory(@site, base_directory.to_s.dup)
+      @base_directory = derive_base_directory(
+        @site, base_directory.to_s.dup
+      )
     end
 
     def base_directory
       @base_directory.to_s
     end
 
     def derive_base_directory(site, base_dir)
-      if base_dir.start_with?(site.source)
-        base_dir[site.source] = ""
-      end
+      base_dir[site.source] = "" if base_dir.start_with?(site.source)
       base_dir
     end
 
     def relative_to_source(entry)
-      File.join(base_directory, entry)
+      File.join(
+        base_directory, entry
+      )
```

Comment:
@envygeeks My argument is: keep pull request diffs to content changes because it's easier to review. I don't think that's facetious at all – if you commit style changes to master, so be it, but don't include the style changes in pull requests if you can help it.

I'm going to fix up rubocop errors soon and add that to the CI pipeline so that style changes can be handled by Travis.